src/shared/domain/definition/case-field.model.ts

```diff
@@ -16,6 +16,7 @@ export class CaseField implements Orderable {
   hint_text?: string;
   security_label?: string;
   display_context: string;
+  display_context_parameter?: string;
```

Comment:
Why are we introducing new concept of display_context if we already have one?

Reply:
This implementation of backend change as per RDM-3530